gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/JobStatus.java

```diff
@@ -0,0 +1,118 @@
+package com.google.gcloud.bigquery;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Lists;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * A Google BigQuery Job status. Objects of this class can be examined when polling an asynchronous
+ * job to see if the job completed.
+ */
+public class JobStatus implements Serializable {
+
+  private static final long serialVersionUID = -714976456815445365L;
+
+  /**
+   * Possible states that a BigQuery Job can assume. A {@link State#PENDING} job is waiting to be
```

Comment:
Here too it would be nice to have the javadoc per enum value.

Also, s/suceeding/succeeding/